public/js/graph/react_graph.js

```diff
@@ -1,5 +1,22 @@
 import * as tooltip from 'es6!graph/tooltip';
 
+
+/**
+ * 
+ * Search for target node in list of nodes,
+ * or if node not found search through list of bools.
+ * @param {Node} targetNode
+ * @returns {Node}
+ *
+ */
+
+function refLookUp(targetNode) {
```

Comment:
Generally try not to use `this` when there isn't a clear class.

In this particular case, you can probably just pass in `svg` as an argument.